apps/src/code-studio/teacher.js

```diff
@@ -0,0 +1,39 @@
+/* globals dashboard  */
+/**
+ * JS used by _teacher.html.haml
+ */
+import $ from 'jquery';
+
+function resizeScrollable() {
+  var newHeight = $('.teacher-panel').innerHeight() -
+      $('.teacher-panel h3').outerHeight() -
+      15 - // magic..
+      $('.non-scrollable-wrapper').outerHeight();
+  $('.scrollable-wrapper').css('max-height', newHeight);
+}
+
+$(document).ready(() => {
+  $(window).resize(dashboard.utils.debounce(resizeScrollable, 250));
```

Comment:
Moving this into JS means that all of our usages of dashboard.utils are now in the JS pipeline. The only things in utils are `debounce` and `throttles`, and both just map to lodash methods. We could, at this point, replace all usages with lodash directly. However, doing so in this particular case would mean adding all of lodash to this bundle. Could be lodash is small enough that we don't really care about the duplicated download?